packages/components/src/components/Card/Card.tsx

```diff
@@ -63,13 +65,14 @@ type CardContainerProps = {
     $isClickable: boolean;
     $variant?: CardVariant;
     $hasLabel: boolean;
+    $overflow: TransientAllowedFrameProps['$overflow'];
 };
 
 const CardContainer = styled.div<CardContainerProps>`
     position: relative;
     border-radius: ${borders.radii.md};
     cursor: ${({ $isClickable }) => ($isClickable ? 'pointer' : 'default')};
-    overflow: hidden;
```

Comment:
actually I think it couldn't work before

Reply:
Well, for card container `overflow:hidden` make sense glboaly.. I'd say that graph card should be an exception with `overflow: visible`.

So, I'd think about ti before we change it globally.

Reply:
Yes, I agree with you. And it should be like that after my changes. If you don't fill `overflow` prop component will use `overflow: hidden`, but at least now it's possible to change it.